library/ecp.c

```diff
@@ -3050,11 +3601,17 @@ int mbedtls_ecp_check_privkey( const mbedtls_ecp_group *grp,
             return( 0 );
     }
 #endif /* MBEDTLS_ECP_SHORT_WEIERSTRASS_ENABLED */
+#if defined(MBEDTLS_ECP_EDWARDS_ENABLED)
+    ( void ) d;
+    /* FIXME: add a check for Edwards */
```

Comment:
Comment from @mpg: https://github.com/Mbed-TLS/mbedtls/pull/3245#discussion_r490831276

Note: I'll need to check the references in more depths, but it might be that we have nothing to check here, except the size of the key (no more than x bytes).